src/public/js/review-configuration.js

```diff
@@ -2,20 +2,26 @@ const ipcRenderer = require('electron').ipcRenderer;
 const remote = require('electron').remote;
 const basepath = remote.app.getAppPath();
 
-
 window.onload = function(){
     document.getElementById('reviewButton').onclick = getReviewConfiguration;
     document.getElementById('cancelButton').onclick = closeWindow;
-    $('#output').val(`${basepath}/cntk`);
+    $('#output').val(`${basepath}/review`);
```

Comment:
So..basebath is the app path, right? I wonder if it won't be better to set it as the directory that the video file is in. Since you already save the .json file next to the video file it might make more sense. Then again, very not critical.